src/api/youtube-api.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import { YOUTUBE_SEARCH_API_URL } from './endpoints';
+import { YOUTUBE_API_KEY } from './api-keys';
+
+export const fetchSearchResults = async query => {
+  try {
+    return await axios.get(YOUTUBE_SEARCH_API_URL, {
+      params: {
+        part: 'snippet',
+        q: query,
+        key: YOUTUBE_API_KEY,
+        type: 'video',
+      },
+    });
+  } catch (err) {
```

Comment:
2 types
javascript. -> error.message
server -> error.response.message
// usually we return error